kelpy/deployment.py

```diff
@@ -40,6 +41,30 @@ def get(client, name, namespace="default"):
     return response
 
 
+def update(client, name, body, namespace="default", timeout=100):
+    try:
+        response = client.patch_namespaced_deployment(name, namespace, body)
+    except ApiException as e:
+        if e.reason == "Conflict":
+            return False
+        raise e
+
+    w = watch.Watch()
+    for event in w.stream(
+        client.list_deployment_for_all_namespaces, timeout_seconds=timeout
+    ):
+        if (
+            event["type"] == "ADDED"
+            and event["object"].metadata.name == response.metadata.name
+            and event["object"].status.available_replicas
```

Comment:
@ramreddyamanaganti can you please explain why we have this line here, is it not redundant with now with the statement below?

Reply:
@pmyjavec It is the  pre-check for the `event["object"].status.replicas == event["object"].status.available_replicas`.
 Checking `event["object"].status.available_replicas` should not be `None`.